yoti-stub/yoti-spec.yaml

```diff
@@ -0,0 +1,1567 @@
+openapi: 3.0.2
+info:
+  title: Yoti IBV Instructions Service
+  version: 1.0.0
+
+tags:
+  - name: Public Endpoints
```

Comment:
We only have public endpoints defined here, but the /sessions et al all tag themselves with backend endpoints.

Are these tags defined this way for a reason? They all seem to be publicly facing

Reply:
It is the tag from the Instructions.yaml as we haven't received combined openapi spec of Sessions+Instructions endpoints. Yoti stub isn't using it.